runtime/doc/api.txt

```diff
@@ -1449,6 +1479,22 @@ nvim_select_popupmenu_item({item}, {insert}, {finish}, {opts})
 nvim__inspect_cell({grid}, {row}, {col})                *nvim__inspect_cell()*
                 TODO: Documentation
 
+                                                        *nvim_init_mark_ns()*
+nvim_init_mark_ns({namespace})
```

Comment:
The function does not exists.
And `nvim_create_namespace()` API is already added in neovim.

Reply:
`api.txt` is autogenerated. we can just delete this file from the PR.